WordPress/wporg-main-2022
# Remove local navigation block

Closes #322

Depends on https://github.com/WordPress/wporg-mu-plugins/pull/443

### How to test the changes in this Pull Request:

1. Set your mu-plugins to https://github.com/WordPress/wporg-mu-plugins/pull/443
2. Check that dynamic menus can still be selected in the admin
3. Check that the menus on the about sub pages still function as before
